web/src/games/cardtable/game.ts

```diff
@@ -0,0 +1,194 @@
+import { Ctx } from 'boardgame.io';
+import { INVALID_MOVE } from 'boardgame.io/core';
+import { dealCribbage } from './deals';
+
+export interface IG {
+  count: number;
+  deck: ICard[];
+  hands: ITable;
+  stock: ICard[];
+}
+
+export interface ITable {
+  north: IHand;
+  south: IHand;
+  east: IHand;
+  west: IHand;
+}
+
+export interface ICard {
+  id: string;
+  rank: number;
+  faced: boolean;
+  img: string;
```

Comment:
remove this field, it can be derived from id.

Reply:
This one needs to remain as implemented for now -- the game currently has no abstraction for private cards, in my earlier versions of this game the cards were given anonymous identities to help prevent cheating on the client.  The server was the authoritative glue between anonymousID -> cardGlyph and managed privacy filters for each players context.  card.ID will likely become this mechanism. and at that point I don't want to rip a bunch of associations out that calculate that mapping.  If this isn't a strong objecting I ask for some grace on this early development.

Reply:
See discussion above. File paths are not OK here.